Library/Formula/nim.rb

```diff
@@ -1,8 +1,9 @@
 class Nim < Formula
   desc "Statically typed, imperative programming language"
   homepage "http://nim-lang.org/"
+  version "0.13"
   url "http://nim-lang.org/download/nim-0.13.0.tar.xz"
-  sha256 "688da1487eb507e0aa96b757a2a4a311123ad38ab68a10277a0415724f982b33"
+  sha256 "cd61f5e5768d4063596d6df578ae9bb5f9d52430773542987e91050b848cb1a9"
   head "https://github.com/Araq/Nim.git", :branch => "devel"
 
```

Comment:
:+1: to `revision 1`. Also, upstream should be told not to change their source files like that.